crates/aptos-logger/Cargo.toml

```diff
@@ -40,6 +40,7 @@ pretty_assertions = { workspace = true }
 [features]
 default = []
 tokio-console = ["console-subscriber"]
+node-identity = ["aptos-node-identity"]
```

Comment:
should we make this default?  `aptos-node` is the only one to use this I believe, so no need to be default I think. wdyt?

Reply:
Yeah, let's enable this feature only for `aptos-node`. Other crates using `aptos-logger` will get the default which doesn't have a dependency on `aptos-node-identity`